DQM/HcalTasks/python/OfflineSourceSequence_pp.py

```diff
@@ -32,6 +34,17 @@
     recHitPreRecoTask +
     rawTask )
 
+hcalOnlyOfflineSourceSequenceGPU = cms.Sequence(
+    digiTask +
+    recHitTask +
+    rawTask +
+    DQMHcalGPUComparisonTask
+)
+
+from Configuration.ProcessModifiers.gpu_cff import gpu
+gpu.toReplaceWith(hcalOnlyOfflineSourceSequence, hcalOnlyOfflineSourceSequenceGPU)
```

Comment:
Now that #36742  has been merge, could you:
  - introduce `Configuration/ProcessModifiers/python/gpuValidationHcal_cff.py` similar to `Configuration/ProcessModifiers/python/gpuValidationEcal_cff.py` from #36742
  - add `gpuValidationHcal` to `Configuration/ProcessModifiers/python/gpuValidation_cff.py`
  - change these lines to
  ```suggestion
from Configuration.ProcessModifiers.gpuValidationHcal_cff import gpuValidationHcal
gpuValidationHcal.toReplaceWith(hcalOnlyOfflineSourceSequence, hcalOnlyOfflineSourceSequenceGPU)
```

?

Reply:
Didn't realize I could have directly click the commit button from your suggestion, I went a detour to pull that Ecal changes from master and manually added the suggested changes in another commit.

Reply:
Could you rebase the branch to avoid having the merge commit ?

Something like
```bash
# get your branch
git checkout HcalDQM-GPUModule

# get the latest master branch
git fetch official-cmssw master:master

# rebase your branch on top of the latest master branch
git rebase master

# make sure everything still builds fine
scram b -j`nproc`

# force-push the changes to the branch on GitHub 
git push my-cmssw +HcalDQM-GPUModule:HcalDQM-GPUModule
```